config/samples/bpfman.io_v1alpha1_tcx_pass_tcprogram.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: bpfman.io/v1alpha1
+kind: TcxProgram
+metadata:
+  labels:
+    app.kubernetes.io/name: tcxprogram
+  name: tcx-pass-all-nodes
+spec:
+  bpffunctionname: pass
+  # Select all nodes
+  nodeselector: {}
+  interfaceselector:
+    primarynodeinterface: true
+  priority: 0
+  direction: ingress
+  bytecode:
+    image:
+      url: quay.io/bpfman-bytecode/tc_pass:latest
```

Comment:
You can change this to `quay.io/bpfman-bytecode/tcx_test:latest` now with `bpffunctionname: tcx_pass`